src/components/templates/PosterPage/components/ShareModal/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { FACEBOOK_APP_ID } from "secrets";
+import { PosterPageVenue } from "types/venues";
+import { WithId } from "utils/id";
+
+const getCategoriesFromVenue = (venue: WithId<PosterPageVenue>) => {
+  return venue?.poster?.categories || [];
+};
+
+export const getTitleTextForSharing = (
+  venue: WithId<PosterPageVenue>,
+  url: string
+) => {
+  if (!venue.poster) return "Check out this OHBM Poster";
+
+  return `Check out this OHBM Poster, ${venue.poster.title} by ${venue.poster.authorName} at ${url}`;
+};
+
+export const getFacebookHref = (
+  venue: WithId<PosterPageVenue>,
+  url: string,
+  text: string
+) => {
+  const categories = getCategoriesFromVenue(venue);
+  const hashtagsString = categories.reduce((result, hash, index, array) => {
+    const isLast = index === array.length - 1;
+    const separator = isLast ? "" : ", ";
+    return result + `#${hash}${separator}`;
+  }, "");
```

Comment:
[NB] This seems.. convoluted. Is there a simpler way for us to handle this? What's the intended outcome of this code? It looks like we could just do
```
categories.map(category => `#${category}).join(', ')
```
or similar to solve the same need here?

Reply:
Does the sharer accept multiple hashtags in this format? Do we need to url encode the `#`?

Reply:
This seems to be the relevant docs for this feature: https://developers.facebook.com/docs/sharing/reference/share-dialog#redirect

Looking at https://developers.facebook.com/docs/sharing/reference/share-dialog#params_share it seems to imply that only a single hashtag can be included?

Reply:
We could also potentially use `PLATFORM_BRAND_NAME` from `src/settings` as the hashtag? This would work for `env/ohbm` (which is set to `OHBM2021`), in most other environments it will just be set to `Sparkle`.

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/7fd77d97e2c82c75f9dfb0f9295fe01606cae835